lib/scala/logging-service-logback/src/main/java/org/enso/logging/service/logback/SocketLoggingNode.java

```diff
@@ -31,15 +32,21 @@ public class SocketLoggingNode implements Runnable {
   SocketAddress remoteSocketAddress;
 
   Logger logger;
-  boolean closed = false;
+  // 0 - not started
+  // 1 - running
+  // 2 - closing
+  // 3 - closed
+  volatile int state = 0;
```

Comment:
why not enum?

Reply:
Just easier to compare. But i can also use `ordinal` on enum, sure.